python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ptp.j2

```diff
@@ -37,6 +37,9 @@ ptp mode {{ ptp.mode }} one-step
 ptp mode {{ ptp.mode }}
 {%         endif %}
 {%     endif %}
+{%     if ptp.profile is arista.avd.defined %}
```

Comment:
for me the config order as below, so if this is same in your platform then move this logic just before to ptp ttl
```
ptp clock-identity 11:11:11:ff:ff:11:11:11
ptp domain 17
ptp message-type event dscp 46 default
ptp message-type general dscp 36 default
ptp mode boundary one-step
ptp priority1 101
ptp priority2 102
ptp profile g8275.1    <--------------
ptp ttl 12
ptp forward-unicast
```